### PR TITLE
Start script improvements

### PR DESCRIPTION
**Pull request by hiltonlima**
- Removed "version" from docker compose files
- Added message in case of failure to start the container

# Message when backend fails
image

### CODE REVIEW

**Review by antonio-amjr — APPROVED** [review]

File: scripts/start.sh [hunk 1]
```diff
@@ -69,7 +99,7 @@ else
     until docker compose exec frontend curl --fail -s --output /dev/null http://localhost:4200
     do
         echo -n "."
-	sleep 5
+	    sleep 5
```
Comment (ccruzagralopes) [inline scripts/start.sh:102]: Was this change intentional?

**Review by ccruzagralopes — APPROVED** [review]

**Review by rquidute — APPROVED** [review]

File: scripts/start.sh (hunk 1 above)
Comment (hiltonlima) [reply scripts/start.sh]: Fixed. Tab character replaced by spaces
